generators/app/index.js

```diff
@@ -145,6 +159,13 @@ module.exports = class extends BaseGenerator {
         this.skipCheckLengthOfIdentifier = this.configOptions.skipCheckLengthOfIdentifier =
             this.options['skip-check-length-of-identifier'] || this.config.get('skipCheckLengthOfIdentifier');
         this.jhiPrefix = this.configOptions.jhiPrefix = _.camelCase(this.config.get('jhiPrefix') || this.options['jhi-prefix']);
+
+        this.entitySuffix = this.configOptions.entitySuffix =
+            this.config.get('entitySuffix') === false ? '' : this.config.get('entitySuffix') || this.options['entity-suffix'];
+
+        this.dtoSuffix = this.configOptions.dtoSuffix =
+            this.config.get('dtoSuffix') === false ? '' : this.config.get('dtoSuffix') || this.options['dto-suffix'];
```

Comment:
Same as my previous comment

Reply:
OK, just to be sure... Only the `false` check is not required anymore ?
The fix has been provided inside jhipster-core ?

Reply:
Just to let you know...

I can't use this in Javascript : `this.config.get('dtoSuffix') || this.options['dto-suffix']` because if `this.config.get('dtoSuffix')` is an empty string, it's a falsy value, thus the `this.options['dto-suffix']` is applied.

I need to do this : 
```javascript
this.dtoSuffix = this.configOptions.dtoSuffix = _.isNil(this.config.get('dtoSuffix'))
            ? this.options['dto-suffix']
            : this.config.get('dtoSuffix');
```
(`_.isNil` is true if the value is null or undefined)